kclvm/compiler_base/3rdparty/rustc_errors/src/lib.rs

```diff
@@ -0,0 +1,362 @@
+//! Text rendering and related helper functions.
+//!
+//! Reuse 'styled_buffer.rs' in 'rustc_errors',
+//! and 'styled_buffer.rs' has been modified to fit the feature of 'Compiler-Base'.
+//!
+//! - add method 'appendl()' and 'putl()' to 'StyledBuffer'.
```

Comment:
`appendln` and `putln` ?

Reply:
rename putl() to pushs().
rename appendl() to append_last_line().